src/stylesheets/index.css

```diff
@@ -49,16 +45,3 @@ a {
   margin: 0 0 5vmin 0;
   border-bottom: 1px solid #ddd;
 }
-
-/**
- *
- */
-.list-inline {
```

Comment:
By default, Tailwind removes sets `list-style: none;` to every list item. To show some list items inline, all that's needed are a few classes. For example, the HTML below would display list items next to each other with 1rem of spacing:

```html
<ul class="flex space-x-4">
  <li>Item 1</li>
  <li>Item 2</li>
  <li>Item 3</li>
</ul>